### PR TITLE
feat: Add clients for actor env vars

### PR DESCRIPTION
**Pull request by jirimoravcik**
This PR adds clients for the new env vars API.
I added TODOs to the docs links, because as of now, the docs are not created.

@mnmkng, What is the suggested procedure here - create docs first and then merge at the same time or can we just release new client and add the docs later?

### CODE REVIEW

**Comment by mnmkng** [comment]
@jirimoravcik Looks like the tests are failing for some reason. Otherwise it looks good. We can definitely release the new client even before we add the API docs. The question is, Client v2 is not stable yet, so if you need this released fast, we should merge it and then backport it to the `version/1.x` branch and release from there.

**Comment by jirimoravcik** [comment]
> Weird. I think they passed on the last commit to master. @vladfrangu can you please check this when you have a moment?

Yeah, but the weird thing is that this repository doesn't have a `package-lock.json` file, so that's the cause 99%. Is there a reason for that?

**Comment by B4nan** [comment]
Finally a man of manners! Yes please, we should use lock files (together with lockfile maintenance from renovate) to actually find those issues early and have it under control (plus the benefit of caching node modules and making CI faster).

I can confirm head/master is failing the same way. Can't really tell whether we should adjust the assertions or if it's important for us to have the `;charset=utf-8` suffix in content type header.

**Comment by vladfrangu** [comment]
It's an update to axios (or a downstream module in it) that's causing this issue. Pinning it to the version we're on right now in package.json "solves" it, but the better question is the one @B4nan just made

**Comment by mnmkng** [comment]
Well if we had the deps locked, we would've not learned about the problem this fast, would we? And all the users would be installing a broken library. Thanks to not having the deps locked, we learned about the issue immediately. But let's leave this discussion for some other time. For now, just fix the problem.

**Comment by vladfrangu** [comment]
So then what's the intended fix? Drop the suffix (is it even necessary), or pin axios to a version?

**Comment by B4nan** [comment]
> Well if we had the deps locked, we would've not learned about the problem this fast, would we? 

That is where the lockfile maintenance steps in, we would know it as there would be a PR telling us the lockfile update failed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2.1.0 / TBD
+===================
+- Added `ActorEnvVarCollectionClient` and `ActorEnvVarClient`
+
```
Comment (B4nan) [inline CHANGELOG.md:4]: 2.0.0 is not out yet, so move this to that version

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "apify-client",
-	"version": "2.0.1",
+	"version": "2.1.0",
```
Comment (B4nan) [inline package.json:3]: no need for version bump as 2.0.0 is not yet out

**Review by B4nan — CHANGES_REQUESTED** [review]
fixed the assertions in master and merged it here, so things should be passing

left few comments, not sure what to do with the docs?

File: package.json (hunk 2 above)
Comment (mnmkng) [reply package.json:3]: 2.0.0 is out on NPM as far as I know.

Comment (vladfrangu) [reply package.json:3]: The beta is
image

Comment (mnmkng) [reply package.json:3]: It's not tagged as it was published by accident, but it's there 
image

**Comment by jirimoravcik** [comment]
@B4nan @mnmkng @vladfrangu

**Comment by jirimoravcik** [comment]
> i'll be happy to merge if we can make it pass the CI checks (and rebase to remove conflicts)
> 
> also I see some TODOs in there, without actual description/action steps - are those something you should resolved before we merge, or something for others? (who?)

conflicts & pipeline fixed. the todos are there because the documentation has not been created yet. I could just remove the todos?

**Comment by B4nan** [comment]
let's keep them there i guess

**Pull request merged**